schemas/user.py

```
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+from pydantic import BaseModel, EmailStr, model_validator, StringConstraints
+from typing_extensions import Annotated
```

Ruff (0.14.11)

Import from `typing` instead: `Annotated`

Import from `typing`

(UP035)